File: src/main/java/com/model/Location.java
package com.model;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Builder
@Data
@AllArgsConstructor
@NoArgsConstructor
public class Location {

    private int id;
    private String country;
    private String city;
    private String weather;
}

File: frontend/src/app/models/Location.ts
export class Location{
  id: string;
  country: string;
  city: string;
  weather: string;


  constructor(id: string, country: string, city: string, weather: string) {
    this.id = id;
    this.country = country;
    this.city = city;
    this.weather = weather;
  }
}
